TS03-Subset-Data-and-No-Data-Values-In-R.Rmd

```diff
@@ -162,119 +168,171 @@ write.csv(harMet15.09.11, file="Met_HARV_15min_2009_2011.csv")
 
 ```
 
-
+<div id="challenge" markdown="1">
 #Challenge: Subsetting & Plotting
-1.Plot precipitation for only July 2010 in the Harvard Forest
- 
+
+Plot precipitation data for the month of July 2010 in Harvard Forest.
+
+</div>
+
 ``` {r challenge-code-subsetting, include=TRUE, results="hide", echo=FALSE}
+
+#subset out data points from July 2010
 harMet15_July2010 <- subset(harMet15.09.11, datetime >= as.POSIXct('2010-07-01 00:00',
         tz = "America/New_York") & datetime <=
         as.POSIXct('2010-07-31 23:59', tz = "America/New_York"))
 
-#did it work?
+#view first and last rows of data.frame to ensure subset worked
 head(harMet15_July2010$datetime)
 tail(harMet15_July2010$datetime)
 
-#ploting precip in July
+#plot precip data for July 2010
 qplot (datetime, prec,
        data= harMet15_July2010,
-       main= "Precipitation in July 2010\nHarvard Forest",
+       main= "Precipitation: July 2010\nNEON Harvard Forest Field Site",
        xlab= "Date", ylab= "Precipitation (mm)")
 ```
 
 
-## Missing Data - Dealing with data gaps
+## Managing Missing Data: NoData values
 
-###Checking for No Data Values
+###Checking for NoData Values
 In the [Time Series Metadata Lesson ]({{site.baseurl}}/R/Time-Series-Metadata/ "Time Series Metadata")
-we viewed the metadata associated 
-with these data and found out that the missing values are given an NA value. One 
-must always check for missing values in any of the variables with which one is 
-working.  
-
-Do we have missing values in our data set? 
-
-An easy way to check for this is the
-`is.na()` function. By asking for the `sum()` of `is.na()` we can see how many 
-NA/missing values we have. 
+we viewed the metadata for these data and discovered that missing values are designated
+using `NA` - a common NoData value placeholder. 
+
+**Excerpt from the metadata:** `airt: average air temperature. Average of daily 
+averages. (unit: celsius / missing value: NA)`
+
+If we are lucky when working with external data, the `NoData` value is clearly specified
+in the metadata. Sometimes this value is `NA` or `nan` (not a number). However, 
+`NA` isn't always used. Text values can make data storage difficult for some 
+programs and thus, sometimes you'll encounter a large negative value such as `-9999`
+used as the `noData` value. Othertimes, we might see blank values in a data file
```

Comment:
This paragrpah has 2 instances of `NoData` and 4 of `noData`.   I looked this up for another lesson I believe we went with  NoData  but I'm not 100% sure right now.  (ERSI uses NoData not noData).  Lets make them all consistent.